stackslib/src/chainstate/nakamoto/tests/mod.rs

```diff
@@ -1636,3 +1643,272 @@ pub fn test_get_highest_nakamoto_tenure() {
     assert_eq!(highest_tenure.tenure_index, 3);
     assert_eq!(highest_tenure.num_blocks_confirmed, 10);
 }
+
+/// Test that we can generate a .miners stackerdb config.
+/// The config must be stable across sortitions -- if a miner is given slot i, then it continues
+/// to have slot i in subsequent sortitions.
+#[test]
+fn test_make_miners_stackerdb_config() {
+    let test_signers = TestSigners::default();
+    let mut peer = boot_nakamoto(
+        function_name!(),
+        vec![],
+        test_signers.aggregate_public_key.clone(),
+    );
+
+    let miner_keys: Vec<_> = (3..13).map(|_| StacksPrivateKey::new()).collect();
```

Comment:
Why is `3..13` being used? That seems like a very specific range, but I don't understand why.

Reply:
At some point there was special behavior for `i < 3`, but it no longer appears to be there.  It's fine to change this all to `0..10`, and remove any `i - 3`'s in the loops that follow (as well as any other place where `i` is compared to `3`).

Reply:
Will do!